linux_os/guide/system/apparmor/all_apparmor_profiles_enforced/ansible/shared.yml

```diff
@@ -0,0 +1,20 @@
+# platform = multi_platform_sle
+# reboot = false
+# strategy = restrict
+# complexity = low
+# disruption = low
+
+- name: Ensure all AppArmor Profiles are reloaded
+  ansible.builtin.shell: apparmor_parser -q -r /etc/apparmor.d/
```

Comment:
Please, use the `command` module instead of `shell` when the `command` is enough.